core/src/main/scala/ai/h2o/sparkling/backend/utils/H2OContextExtensions.scala

```diff
@@ -0,0 +1,62 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.backend.utils
+
+import java.io.File
+
+import org.apache.spark.h2o.H2OContext
+import water.api.ImportHiveTableHandler
+import water.api.ImportHiveTableHandler.HiveTableImporter
+import water.fvec.Frame
+
+trait H2OContextExtensions extends H2OContextUtils {
```

Comment:
Is the only place where `H2OContextUtils` are used. if so, couldn't they be merged with this class

Reply:
The motivation for the separation was to keep public methods in extensions which are user facing, and in utils, just utilities. Bud I guess at this moment this is not necessary. Merging together 👍